modules/nw-ovn-kubernetes-rollback.adoc

```diff
@@ -27,26 +29,82 @@ $ oc annotate Network.operator.openshift.io cluster \
   'networkoperator.openshift.io/network-migration'=""
 ----
 
-. To change the default CNI network provider, enter the following command:
+. To stop all of the machine configuration pools, enter the following commands:
```

Comment:
These steps are similar to the other module. If you take those suggestions, please apply them here, too.